.github/pull_request_template.md

```diff
@@ -9,7 +9,7 @@ Fixes #
 
 `[Author TODO: add description of changes.]`
 
-- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).
+- [ ] This functionality can be added in [`rust-vmm`][1].
```

Comment:
For long links, this is the correct way of formatting this in markdown. Since we are adding a long link, we need to be consistent with all link usage in the doc.